internal/testplanet/satellite.go

```diff
@@ -154,7 +154,7 @@ func (planet *Planet) newSatellites(count int) ([]*satellite.Peer, error) {
 			Repairer: repairer.Config{
 				MaxRepair:    10,
 				Interval:     time.Hour,
-				Timeout:      10 * time.Second, // Repairs can take up to 4 seconds. Leaving room for outliers
+				Timeout:      1 * time.Minute, // Repairs can take up to 10 seconds. Leaving room for outliers
```

Comment:
The question is whether the flakiness was fixed by the go test parallelism changes or by this change?

Reply:
We added the logger last week to narrow down the cause of the tests when they failed (before the parallelism changes), and the cause of the failing was always due to repair being cutoff because of the 10s timeout (which we didn't see before because the repairer logs were not visible in jenkins).
After increasing the timeout, we were able to get the test to always pass. I believe 10s was the timeout when the testplanet test had fewer storagenodes, so it makes sense that it is not always sufficient now.

Reply:
From the logs it seems that the tests started passing after merging in the parallelism changes. So is the 10s is at the cusp of passing or was something else going on?

Reply:
The tests were also consistently passing before jenkins started failing a lot (for reasons I think unrelated to this change). I believe 10s was on the cusp before. This test was failing on like 50% of builds before. We can set it back to 10s and see what happens to the builds if you'd like

Reply:
Yeah, would be nice to know how much difference that made. But yeah, 1min is fine as well if it makes sense for the test.

Reply:
We tried it a bunch of times again with 10s and it seemed to work most of the time - but we did see the timeout error one time over the course of 10 builds. I think it is dependent on how many simultaneous builds are going on and how much load Jenkins is under. 1 minute shouldn't be necessary, but it will prevent the test from failing frequently like it has been lately.

Reply:
Jenkins doesn't run multiple jobs concurrently, so the load should be similar. There are slight differences between the jenkins nodes though.